src/tchannel_bridge.js

```diff
@@ -0,0 +1,103 @@
+// @flow
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import * as crossdock_constants from '../crossdock/src/constants';
+import * as constants from './constants';
+import Int64 from 'node-int64';
+import Span from './span';
+import SpanContext from './span_context';
+import Utils from './util';
+import tchannelSpan from './vendor/tchannel/span';
+import Tracer from '../src/tracer';
+
+export default class TChannelBridge {
+    static toTChannelSpan(span: Span): TChannelSpan {
+        return new tchannelSpan({
+            id: Utils.tchannelBufferToIntId(span.context().spanId),
+            traceid: Utils.tchannelBufferToIntId(span.context().traceId),
+            name: span.name,
+            parentid: Utils.tchannelBufferToIntId(span.context().parentId),
+            flags: span.context().flags || 0
+        });
+    }
+
+    static toTChannelv2Span(span: Span): TChannelSpan {
+        return new tchannelSpan({
+            id: span.context().spanId,
+            traceid: span.context().traceId,
+            name: span.name,
+            parentid: span.context().parentId,
+            flags: span.context().flags || 0
+        });
+    }
+
+    static getSpanFromTChannelRequest(tracer: Tracer, request: any, headers: any = {}, options: any = {}): Span {
+        let traceContext;
+        if (crossdock_constants.TCHANNEL_HEADER_TRACER_STATE_KEY in headers) {
```

Comment:
I am not seeing this constant in https://github.com/uber/jaeger-client-node/blob/master/crossdock/src/constants.js. Where is it defined?